### ISSUE TITLE
Add configurable run parameters to RESTful API

### ISSUE THREAD

**Issue by kitcambridge** [loads/loads-broker#16]
When triggering a run via the RESTful API, several parameters should be allowed:
1) The Run ID
2) DNS names (ie, for this run, the testers should see this DNS name as this set of IP's)

Something like:

```
HTTP POST /api/orchestrate/STRATEGY_ID

dns_names={"testcluster.mozilla.org": [192.16.2.1, 192.16.2.2]}
run_id=MY_AWSOME_ID
```

I like frogs. :frog:

**Comment by bbangert** [comment]
Fixed in c93ef4dca9f9e170f8961e97424aa179033bf596, will add tests before closing this.

**Comment by bbangert** [comment]
Realized that I did not add the dns_names option.
